data/mods/Magiclysm/worldgen/regional_overlay.json

```
@@ -3,6 +3,7 @@
     "type": "region_overlay",
     "//id": "magiclysm_buildings_overlay",
     "regions": [ "all" ],
-    "city": { "shops": { "magic_shop": 100, "used_bookstore": 225 }, "basements": { "magic_basement": 50 } }
+    "city": { "shops": { "magic_shop": 100, "used_bookstore": 225 }, "basements": { "magic_basement": 50 } },
+    "other": { "f_glow_boulder": 0.3333 }
```

That's incorrect - `other` node should be inside of `field_coverage` node:

```suggestion
    "field_coverage":  { "other": { "f_glow_boulder": 0.3333 } }
```

---

How would I fix this, since it's already been merged?

---

In a separate PR like this - https://github.com/CleverRaven/Cataclysm-DDA/pull/35834

---

Wait so you're already doing it? I'm confused.